nncf/torch/exporter.py

```diff
@@ -128,40 +128,37 @@ def _export_to_onnx(self, save_path: str, opset_version: int) -> None:
         original_device = get_model_device(self._model)
         model = self._model.eval().cpu()
         input_tensor_list = []
-        for info in self._model.input_infos:
+        for info in self._model.nncf.input_infos:
             single_batch_info = copy(info)
             input_shape = tuple([1] + list(info.shape)[1:])
             single_batch_info.shape = input_shape
             input_tensor_list.append(create_mock_tensor(single_batch_info, 'cpu'))
 
-        original_forward = model.forward
+        full_arg_forward = model.nncf.get_original_forward()
```

Comment:
The original forward function can now be retrieved using a well-defined function. The `.forward` on the NNCFNetwork function will always be an NNCF-wrapped `forward` version.